deps/build.jl

```diff
@@ -43,7 +43,7 @@ ldflags = ""
 if is_apple()
     ldflags = "$ldflags -undefined suppress -flat_namespace"
 end
-cflags = "-DCOPYAMATRIX -DLONG -DUSE_LAPACK -DCTRLC=1"
+cflags = "-DCOPYAMATRIX -DDLONG -DUSE_LAPACK -DCTRLC=1"
```

Comment:
The CFLAGS here only affect linux.

Reply:
This change breaks linux, I'm reverting it.

Reply:
Ok, in that case the change will have to be made again for the next release of SCS, which I'll probably make in the next day or two, since this change is in the main trunk.